react/features/overlay/components/ReloadButton.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+
+import { translate } from '../../base/i18n';
+
+import { reconnectNow } from '../functions';
```

Comment:
There're 3 import groups (technically, 4 while current non-React Web is still around but that's temporary):

1. Outside/third-party imports.
LF
2. Other features.
LF
3. Files from inside the feature.